packages/openapi-spec/src/openapi-spec-v2.ts

```diff
@@ -282,7 +282,7 @@ export interface SecuritySchemeObject extends Extensible {
   /**
    * The location of the API key. Valid values are "query" or "header".
    */
-  'in'?: 'query' | 'header';
+  in?: 'query' | 'header';
```

Comment:
Interesting, I don't know that we can use `in` keyword as object key without quotation.